docs/devs/e2e-tests.md

```diff
@@ -185,10 +185,8 @@ We observe that the test structure is broken down between a **setup**, the actua
 
 **Test Case**: The core of each test case follows the _Given-When-Then_ format (commented out here as Setup-Replay-Verification). We highly recommend this structured approach as it clearly delineates the setup of the test environment ("Setup"), the end-user actions performed ("Replay"), and the assertion of outcomes ("Verification"). In our example:
 
-- **Setup**: Navigating to the lab order form, adding a new lab order, and saving it.
-- **Replay**: Navigating to the SENAITE HIS component and searching for the client by name.
-- **Verification**: Verify that the client's name is visible in the clients list.
+- **Setup**:
```

Comment:
@Ruhanga  for the test case, it's setup is in the _beforeEach()_,  what could be the appropriate explanation regarding this part?

```
 // setup
  homePage = new HomePage(page);
```

Reply:
> It is not always going to be the same setup for the different test cases
> However it's also good to have common/reused code across all test-cases exist in the beforeEach() function.

This is very true.